packages/web/docs/src/content/self-hosting/changelog.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Self Hosting Changelog
+sidebarTitle: Changelog
+description: Changelog for self-hosting Hive
+---
+
+import fs from 'node:fs/promises'
+import path from 'node:path'
+import { compileMdx, MDXRemote } from '@theguild/components/server'
+
+# Self-Hosting Changelog
+
+Here you can find all the recent improvements and updates about the self-hosting images. The Docker
+images published to the GitHub registry using the same tags. For a high-level overview of
+improvements please refer to [our product updates](/product-updates).
+
+export const source = await fs.readFile(path.resolve('../../../deployment/CHANGELOG.md'), 'utf8')
+
+<MDXRemote compiledSource={await compileMdx(source.replace('# hive', ''))} />
```

Comment:
Should we switch to something like

```suggestion
<MDXRemote compiledSource={await compileMdx(source.replace(/#\s+.*\n/, ''))} />
```

so it still works if someone oblivious uppercases the "Hive" in that heading or changes the word?